client/packages/system/src/Patient/PatientView/PatientView.tsx

```diff
@@ -199,7 +199,7 @@ const PatientDetailView = ({
       };
 
   const { JsonForm, saveData, isSaving, isDirty, validationError } =
-    useJsonForms(
+  useJsonFormsHandler (
```

Comment:
Revert whitespace change (If you have Prettier set up, this should happen automatically)